tests/testapp/tests/test_uuid_utilities.py

```diff
@@ -52,39 +65,109 @@ def test_creating_same_instance_ID_model(self):
         self.assertEqual(firstIDModel, secondIDModel)
         self.assertEqual(InstanceIDModel.objects.count(), 1)
 
-    def test_creating_different_instance_ID_model(self):
-        # change system state
-        with mock.patch('platform.platform', return_value='platform'):
-            with mock.patch('uuid.getnode', return_value=9999999999999):  # fake (random) address
-                (IDModel, _) = InstanceIDModel.get_or_create_current_instance()
-        self.assertEqual(InstanceIDModel.objects.count(), 2)
-        self.assertEqual(IDModel.node_id, '')  # assert that node id was not added
-        self.assertEqual(IDModel.id, InstanceIDModel.objects.get(current=True).id)
-
     def test_only_one_current_instance_ID(self):
-        with mock.patch('platform.platform', return_value='platform'):
+        with mock.patch("platform.platform", return_value="platform"):
             InstanceIDModel.get_or_create_current_instance()
         self.assertEqual(len(InstanceIDModel.objects.filter(current=True)), 1)
 
     def test_same_node_id(self):
-        with mock.patch('uuid.getnode', return_value=67002173923623):  # fake (random) address
+        with mock.patch(
+            "uuid.getnode", return_value=67002173923623
+        ):  # fake (random) address
             (IDModel, _) = InstanceIDModel.get_or_create_current_instance()
             ident = IDModel.id
 
-        with mock.patch('uuid.getnode', return_value=69002173923623):  # fake (random) address
+        with mock.patch(
+            "uuid.getnode", return_value=69002173923623
+        ):  # fake (random) address
             (IDModel, _) = InstanceIDModel.get_or_create_current_instance()
 
-        with mock.patch('uuid.getnode', return_value=67002173923623):  # fake (random) address
+        with mock.patch(
+            "uuid.getnode", return_value=67002173923623
+        ):  # fake (random) address
             (IDModel, _) = InstanceIDModel.get_or_create_current_instance()
 
-        self.assertFalse(InstanceIDModel.objects.exclude(id=ident).filter(current=True).exists())
+        self.assertFalse(
+            InstanceIDModel.objects.exclude(id=ident).filter(current=True).exists()
+        )
         self.assertTrue(InstanceIDModel.objects.get(id=ident).current)
 
```

Comment:
The two tests below are new, and are designed to ensure that the Instance ID doesn't change when we aren't wanting it to.